lib/index.js

```diff
@@ -140,7 +142,8 @@ export default function runContentfulExport (usageParams) {
       task: (ctx) => {
         fs.existsSync(opts.exportDir) || fs.mkdirSync(opts.exportDir)
         ctx.responseFile = `${opts.exportDir}/contentful-export-${ctx.clients.source.spaceId}-${Date.now()}.json`
-        return fs.writeFileSync(ctx.responseFile, jsonStringifySafe(ctx.data, null, 4))
+        fs.writeFileSync(ctx.responseFile, jsonStringifySafe(ctx.data, null, 4))
+        return ctx
```

Comment:
Shame on me, sorry for that

![](https://media.giphy.com/media/d7fTn7iSd2ivS/giphy.gif)